test/example2.proto

```diff
@@ -17,3 +17,15 @@ enum AEnum2 {
 message refexample3 {
   Funky funky = 1;
 }
+
+service MyService {
```

Comment:
Added a service here to validate custom hack namespace is reflected in generated service definitions as well.